templates/al2023/provisioners/install-neuron-driver.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+if [ "$ACCELERATOR_VENDOR" != "neuron" ]; then
+  exit 0
+fi 
+
+PARTITION=$(imds "/latest/meta-data/services/partition")
+
+if [ "$PARTITION" == "aws-iso" ] || [ "$PARTITION" == "aws-iso-b" ]; then
+  echo "Neuron repository not vailable in isolated regions"
+  exit 0
```

Comment:
I think we want the build to fail in this case, so you don't end up with an AMI that looks like it contains the neuron bits, but doesn't.